consensus/block_validation.go

```diff
@@ -71,6 +71,10 @@ func (dv *DefaultBlockValidator) ValidateSemantic(ctx context.Context, child *ty
 
 // ValidateSyntax validates a single block is correctly formed.
 func (dv *DefaultBlockValidator) ValidateSyntax(ctx context.Context, blk *types.Block) error {
+	// hack validate genesisblock
```

Comment:
Unrelated to these changes -- we will probably want some special handling for genesis block validation.